techniques/silverlight/SL11.html

```diff
@@ -74,7 +74,7 @@ <h3>Pausing a decorative animation</h3>
     						is the event handler for the UI control (a button). The event handler
     						retrieves the animation definition from the page resources, and calls
     						the Pause method on the animation. </p>
-            <pre xml:space="preserve">       public MainPage()
+            <pre>       public MainPage()
```

Comment:
```suggestion
            <pre><code>       public MainPage()
```
and add matching `</code>`